server/api/cron.go

```diff
@@ -97,13 +97,13 @@ func PostCron(c *gin.Context) {
 		Branch:    in.Branch,
 	}
 	if err := cron.Validate(); err != nil {
-		c.String(400, "Error inserting cron. validate failed: %s", err)
+		c.String(http.StatusBadRequest, "Error inserting cron. validate failed: %s", err)
```

Comment:
Validation should result in 422 error (400 should be when data is unparsable etc)